### PR TITLE
#14 added configuration file for "Prettier" formatter

### LINKED ISSUE #14: Create a config file adding formatting and linting settings

### PR DESCRIPTION
**Pull request by Anjali0407-git**
# Pull Request Template

**Fixes #14**

### What was changed?

Added config file named as` .prettierrc` for Prettier formatter 

### Why was it changed?

To ensure all developers sticks to same formatter and linting settings

### How was it changed?

Added a new file `.prettierrc` composing all the settings

### CODE REVIEW

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
You're on track, but there's more to do. Let's do this in three parts so it's clear what is being changed by each step. 

1. **Tooling:** now that you have a prettier config you should probably add prettier to your dev dependencies list so anyone can use it run `npx prettier . --check` locally. At that point the basic tools required for linting are in place. You can add this to this PR.
2. **Styling:** in a second PR we can put in the results of running `npx prettier . --fix`. This will automate the first pass of styling. This will also make lots of updates, and we will want to make sure there are only code style updates when we review it.
3. **Automation:** in third PR let's update the `linting` job in the GitHub Actions so that it checks linting on every PR. Once this is done the overall update to linting is in place.

By breaking this work into three separate PRs it's clear what is doing what, and it makes reviewing each piece much easier.

**Comment by Anjali0407-git** [comment]
I have addressed Tooling updates in this PR and created a new [PR](https://github.com/oss-slu/material-donor-mutual-assist/pull/21) addressing Styling updates.
I will read and work on automation part tomorrow.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
-  "dependencies": {
```
Comment (kungfuchicken) [inline package.json:2]: Why is this removed for linting?

File: README.md
Comment (kungfuchicken) [inline README.md:1]: I guess this makes sense to add because you need a place to put how to run prettier?

**Review by kungfuchicken — COMMENTED** [review]
I have a couple questions that need clarification. I don't think anything needs to change necessarily, i just want to understand.

File: package.json (hunk 1 above)
Comment (Anjali0407-git) [reply package.json:2]: This is unwanted code added long ago. We have this dependency in `server/package.json` file. This dependency do nothing at the root directory.

File: README.md
Comment (Anjali0407-git) [reply README.md:1]: Yes exactly.

**Review by kungfuchicken — APPROVED** [review]
lgtm;
